tests/index.test.ts

```diff
@@ -0,0 +1,51 @@
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+const _ = require('lodash/fp')
+const { getInstalledPathSync } = require('get-installed-path')
+const NewRelicLambdaLayerPlugin = require('../src/index')
+
+const serverlessPath = getInstalledPathSync('serverless', { local: true })
+const AwsProvider = require(`${serverlessPath}/lib/plugins/aws/provider/awsProvider`)
+const CLI = require(`${serverlessPath}/lib/classes/CLI`)
+const Serverless = require(`${serverlessPath}/lib/Serverless`)
+const fixturesPath = path.resolve(__dirname, 'fixtures')
+
+describe('NewRelicLambdaLayerPlugin', () => {
+    const stage = 'dev'
+    const options = { stage }
+
+    describe('run', () => {
```

Comment:
idea taken from https://github.com/davidgf/serverless-plugin-canary-deployments 

you test input and output after plugin process input

Reply:
This is really interesting--thanks for including it!